### PR TITLE
don't overwrite added_at when interning a keep_to_user

### PR DESCRIPTION
**Pull request by camhashemi**
@ryanpbrewster @leogrim this leads to some weird behavior on the feed, where when we re-ingest keeps we overwrite `ktu.added_at` to be the later time, even though the `ktu` already existed. I'd prefer `added_at` to be equal to `kept_at` if you're the owner of the keep, less tricky to reason about.

### CODE REVIEW

**Comment by ryanpbrewster** [comment]
I agree with you. Be sure to chat with @eishay about it, this behavior was specifically requested by him.

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepToUserCommander.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ class KeepToUserCommanderImpl @Inject() (
   def internKeepInUser(keep: Keep, userId: Id[User], addedBy: Id[User])(implicit session: RWSession): KeepToUser = {
     ktuRepo.getByKeepIdAndUserId(keep.id.get, userId, excludeStateOpt = None) match {
       case Some(existingKtu) if existingKtu.isActive =>
-        ktuRepo.save(existingKtu.withAddedAt(clock.now))
+        ktuRepo.save(existingKtu)
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepToUserCommander.scala]: If we're not going to mutate it then we don't have to re-save it.

**Comment by camhashemi** [comment]
@eishay what does this grant us? this is the particular bug in question raised by @JRuff42, note the timestamps:

from feed, using earliest `added_at` between `keep_to_library` and `keep_to_user`
screen shot 2016-01-26 at 3 27 07 pm

from keep page, using `kept_at`
screen shot 2016-01-26 at 3 27 38 pm

after this PR, these timestamps would be the same going forward

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepToUserCommander.scala (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/KeepToUserCommander.scala]: :8ball:

**Comment by andrewconner** [comment]
I believe the feed should stay in order (dates should be descending, most recent first). So in this case, this keep was added 6 days ago, so should be with other keeps from 6 days ago...

_I think_, based on the conversation we had yesterday. @leogrim @eishay @JRuff42 please correct me if wrong.

**Comment by eishay** [comment]
@camhashemi could you please attach the link to the task in asana? i want to make sure i'm in sync with Jen

**Comment by camhashemi** [comment]
This has all been in private slack messages. I'm not sure if there's a task for the bug.

Jen and I talked some more, she said that the desired behavior is for the keep to bump up to the top of the feed when a message is added (:+1:). What @ryanpbrewster, @leogrim, and I planned to do was use `keep.last_activity_at` to sort the feed, which gets updated whenever there's an update-worthy event on that keep.

**Comment by eishay** [comment]
@camhashemi talked with Jen, please talk with her tomorrow.

**Pull request merged**